frontend/src/app/components/post-modal/post-modal.component.ts

```diff
@@ -239,6 +239,25 @@ export class PostModalComponent {
     this.comments.push(comment);
   }
 
+  async deleteComment(comment: Comment) {
+    this.dialog.open(ConfirmModalComponent, {
+      width: '500px',
+      data: {
+        title: 'Confirmation',
+        message: 'Are you sure you want to delete this comment?',
+         handleConfirm: () => {
+          // this.socketService.emit(SocketEvent.COMMENT_DELETE, commentID);
```

Comment:
Remove comment please, I think you forgot to move this inside the `canvas.service.ts` function.